src/module/actor/retrograde-map.ts

```diff
@@ -0,0 +1,72 @@
+export const frameToPath: Record<string, string> = {
+    "BALOR": "systems/lancer/assets/retrograde-minis/Retrograde-Minis-Horus-BALOR.png",
```

Comment:
Noticed a few missing here that we have images for:
* Atlas
* Caliban
* White Witch
* Emperor